.github/workflows/develop.yml

```
@@ -291,6 +291,18 @@ jobs:
             kafka: true
             values: "with-remote-wal.yaml"
     steps:
+      - name: Remove unused software
+        run: |
+          echo "Disk space before:"
+          df -h
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /opt/ghc
+          sudo rm -rf /opt/hostedtoolcache/CodeQL
```

Avoid annoying error when these dirs do not exist
```suggestion
          [[ -d /usr/share/dotnet ]] && sudo rm -rf /usr/share/dotnet
          [[ -d /usr/local/lib/android ]] && sudo rm -rf /usr/local/lib/android
          [[ -d /opt/ghc ]] && sudo rm -rf /opt/ghc
          [[ -d /opt/hostedtoolcache/CodeQL ]] && sudo rm -rf /opt/hostedtoolcache/CodeQL
```